interceptors/DataApiInterceptors.cfc

```diff
@@ -125,13 +125,12 @@ component extends="coldbox.system.Interceptor" {
 		}
 	}
 
-	public void function postUpdateObjectData( event, interceptData ) {
-		if ( !_applicationLoaded ) return;
-		dataApiQueueService.queueUpdate( argumentCollection=interceptData );
-	}
-
 	public void function postInsertObjectData( event, interceptData ) {
 		if ( !_applicationLoaded ) return;
+
+		var skipQueue = IsBoolean( interceptData.skipSyncQueue ?: "" ) && interceptData.skipSyncQueue;
```

Comment:
I think @linfeng2011 that this should be `skipDataApiQueue` to be more precise. Can you change it?

Reply:
Hey @DominicWatson , thanks for reviewing. I have change the variable name and added documentation to README accordingly, thanks!